packages/universe-pages/src/initiatives/[id]/page.tsx

```diff
@@ -103,7 +103,7 @@ export default async function Initiative(props: {
         <div className="md:flex md:flex-col items-center">
           <div className="flex flex-col lg:flex-row flex-nowrap gap-10 items-start">
             <div className="w-full lg:w-[60%]">
-              <DonationForm initiative={initiative} />
+              <DonationForm initiative={initiative} rate={rate} />
```

Comment:
rate prop can be removed

Reply:
Same as above. Leave for now, need more testing